cmd/keytransparency-monitor/main.go

```diff
@@ -85,11 +86,14 @@ func main() {
 	store := fake.NewMonitorStorage()
 
 	// Create monitoring background process.
-	mon, err := monitor.NewFromConfig(ktClient, config, signer, store)
+	mon, err := monitor.NewFromDomain(ktClient, config, signer, store)
 	if err != nil {
 		glog.Exitf("Failed to initialize monitor: %v", err)
 	}
-	go mon.ProcessLoop(ctx, *domainID, store.LatestEpoch(), *pollPeriod)
+
+	// TODO(gbelvin): persist trusted roots
+	trusted := trillian.SignedLogRoot{}
+	go mon.ProcessLoop(ctx, *domainID, trusted, *pollPeriod)
```

Comment:
what happened to `store.LatestEpoch()`?

Reply:
`store.LatestEpoch` was only a number.  For consistency proofs, we need to store the full STH.  I'll leave that for a future PR, however, as the monitor is not on the front burner right now.